eng/scripts/generate_aggregate_coverage_pom.py

```diff
@@ -29,6 +29,8 @@
 
 include_groups = []
 
+exclude_projects = ['com.azure.resourcemanager:azure-resourcemanager-samples']
```

Comment:
This exclusion list should not be hard coded. Make this into an argument to the script and update the ci to add this package as an arg when running the script.

Reply:
Sure, sounds good. 
On the other hand, Weidong found out the cause is that `samples` project contains zip/war/jar and jacoco doesn't support them. 
He suggests that rather than excluding the `samples`, we can add
```xml
<includes>
   <include>**/*.class</include>
</includes> 
```
to the jacoco configuration in the pom template (we can do it in this PR). It'll look like:
```xml
  <build>
    <plugins>
      <plugin>
        <groupId>org.jacoco</groupId>
        <artifactId>jacoco-maven-plugin</artifactId>
        <version>{}</version>
        <configuration>
          <outputDirectory>target/site/test-coverage</outputDirectory>
          <includes>
             <include>**/*.class</include>
          </includes>
          <excludes>
             <exclude>META-INF/**</exclude>
          </excludes>
        </configuration>
      </plugin>
    </plugins>
  </build>
```

Either will resolve the issue. Which one do you prefer?